<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamMainWindow</class>
 <widget class="QDialog" name="CamMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdDebugWnd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdRec">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdPlay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdRecSettings">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Record
Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdFPNCal">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Black Cal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdWB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set White
Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdIOSettings">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trigger/IO
Settings</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkFocusAid">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
     width: 28px;
     height: 28px;
}


</string>
   </property>
   <property name="text">
    <string>Focus
Aid</string>
   </property>
  </widget>
  <widget class="QSlider" name="expSlider">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>20</y>
     <width>51</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:vertical {
     border: 1px solid #999999;
     width: 48px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
     margin: 2px 0;
 } 
QSlider::handle:vertical {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 1px solid #5c5c5c;
     height: 60px;
     margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }</string>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblCurrent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblExp">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Shutter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdUtil">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Util</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdBkGndButton">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-20</y>
     <width>241</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
border-style: none;
border-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdDPCButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DPC</string>
   </property>
  </widget>
  <zorder>cmdBkGndButton</zorder>
  <zorder>cmdRec</zorder>
  <zorder>cmdPlay</zorder>
  <zorder>cmdRecSettings</zorder>
  <zorder>cmdFPNCal</zorder>
  <zorder>cmdWB</zorder>
  <zorder>cmdIOSettings</zorder>
  <zorder>chkFocusAid</zorder>
  <zorder>expSlider</zorder>
  <zorder>lblCurrent</zorder>
  <zorder>cmdClose</zorder>
  <zorder>lblExp</zorder>
  <zorder>cmdUtil</zorder>
  <zorder>cmdDebugWnd</zorder>
  <zorder>cmdDPCButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
